crates/autopilot/src/run_loop.rs

```diff
@@ -65,9 +65,11 @@ impl RunLoop {
 
         let mut last_auction = None;
         let mut last_block = None;
+        let mut init_block_timestamp = None;
         loop {
             if let RunLoopMode::SyncToBlockchain = self.synchronization {
                 let block = ethrpc::block_stream::next_block(self.eth.current_block()).await;
+                init_block_timestamp = Some(block.timestamp);
```

Comment:
Other than during the initialization is the block timestamp ever none? If not we should probably use a default value to not have to match on the `Option` unnecessarily.

Reply:
I used an optional value for cases when `RunLoopMode::Unsynchronized` is used to avoid populating inconsistent data. Should become non-optional once `RunLoopMode::Unsynchronized` is removed. Does it make sense?

Reply:
Ah, right. In that case I would put this into the `Unsynchronized` case after the sleep:
```rust
            init_block_timestamp = self.eth.current_block().borrow().timestamp;
```